manager/manager.go

```diff
@@ -163,6 +209,34 @@ func (m *Manager) RunCommand(instanceIds []string, command string) (string, erro
 	return aws.StringValue(res.Command.CommandId), nil
 }
 
+// RunDocument on the given instance ids.
+func (m *Manager) RunDocument(instanceIds []string, name string, parameters []string) (string, error) {
+
+	var params map[string][]*string
+
+	if len(parameters) > 0 {
```

Comment:
These parameters are from the CLI right? I'm thinking that the parsing of the parameters should happen in the CLI, and that this function should just accept a map of parameters directly?

Reply:
Yepp, this is still WIP and it will be refactored.